dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -148,15 +158,46 @@ run_ocp() {
 }
 
 deploy_che_to_ocp() {
-    #Repull init image only if IMAGE_PULL_POLICY is set to Always
-    if [ $IMAGE_PULL_POLICY == "Always" ]; then
-        docker pull "$IMAGE_INIT"
+    #Only generate scripts and config files if deploy_che.sh does not exist in same folder
+    if [ ! -f deploy_che.sh ]; then
+      #Repull init image only if IMAGE_PULL_POLICY is set to Always
+      if [ $IMAGE_PULL_POLICY == "Always" ]; then
```

Comment:
This code gets too complicated. Can you refactor it to have less nesting?

Reply:
Not sure how to do this. I could provide a separate function which would make things more DRY as I repeat this twice in `deploy_che_to_ocp` and `replace_che_stacks_on_ocp`.

Reply:
Yes, make it simpler if possible.

Reply:
Well based on Roman's comment the replace stacks is no longer needed. If someone wants to replace this with custom stacks they can do this via assembly as described in the docs.